### DISCUSSION TITLE
There is something wrong with my import

### DISCUSSION THREAD

**Discussion by Jaseempk** [Cyfrin/foundry-full-course-cu#334] [Q&A] [answered]
A problem is showing-up while writing import at the top of my contract:
```
import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
```
![Screenshot 2023-07-14 161408](https://github.com/Cyfrin/foundry-full-course-f23/assets/99121494/57efa4e8-3bf1-4dfa-9263-f077a3a7791a)

@pacelliv @alymurtazamemon can you please look into this?

**Comment by Teoaks** [comment]
1. Make sure you installed OpenZeppelin using :  ``npm install @openzeppelin/contracts``
2. Make sure your path is correct

**Reply by Jaseempk** [reply]
I have used  `forge` instead of `npm` :
```
forge install OpenZeppelin/openzeppelin-contracts
```
also path is right its on the same directory

**Reply by Jaseempk** [reply]
here is the installation path for reference:
![Screenshot 2023-07-14 165341](https://github.com/Cyfrin/foundry-full-course-f23/assets/99121494/2e2194f5-7b77-4601-9543-e8c0bcc6b8f9)

**Reply by shikhar229169** [reply]
Have you set the remappings correctly?
Consider checking that.

**Reply by Jaseempk** [reply]
Do you mean remapping like this?
```
import "@lib/openzeppelin-contracts/contracts/token/ERC721/ERC721.sol";
````
If YES then even that's not helping

**Reply by ghost** [reply, marked as answer]
@Jaseempk in your contract use this import:

```javascript
import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
```
Then in `foundry.toml` add a remappings **under** `[profile.default]`:

```toml
[profile.default]
# other stuff...

remappings = ['@openzeppelin/contracts=lib/openzeppelin-contracts/contracts']
```

Now... if you do this and after a few seconds the error remains, just run `forge build` I don't know the reason but in some occassions the linter from the formatter extension bothers with the paths but fingers crossed the project should build.

**Reply by Jaseempk** [reply]
`forge build` helped to solve it...thanks mate @pacelliv
